### DISCUSSION TITLE
Custom filter to include/exclude pages

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#774] [Anything Else]
"Would be cool to be able to filter out urls. I have a ton of urls and would like to sometime be able to see stats without some of my subfolders: like I want to see all except everything with '/blog' in the url"

([feedback via twitter](https://twitter.com/brenioux/status/1366645001510354945))

similar request was placed in [a different thread here on github](https://github.com/plausible/analytics/discussions/136#discussioncomment-400893):

"I'd like to see stats of the most visited pages of our Blog, in this case I would filter web pages starting with http://example.com/blog"

similar feature is available in Search Console, GA etc. it could work like this: click "custom filter", select if you want to include/exclude and type what you want.

**Comment by ukutaht** [comment]
A lot of overlap with  #136

**Reply by metmarkosaric** [reply]
i was under impression in that thread that the request is to change the data completely in the database to remove personal data? otherwise they're pretty much the same. this one just cares about a more useful display of data rather than changing the database

**Comment by metmarkosaric** [comment]
the part with tracking blog content itself is now possible using the new pageview goals. see how we do it on our [own live demo](https://plausible.io/plausible.io/pages?period=30d&goal=Visit+%2Fblog**) and read [how you can do it too](https://plausible.io/docs/pageview-goals).

**Comment by arthursoares** [comment]
:+1: for this.

Showing & filtering Pages in a "tree" format, not only by absolute URLs

**Comment by metmarkosaric** [comment]
this is now live. click on the "add filter" button on the top of your dashboard to explore. you can learn more here https://plausible.io/docs/filters-segments#using-the-add-filter-menu-for-extra-functionality
